lua/plugins/packer_update.lua

```
@@ -0,0 +1,57 @@
+  
+  if nvim_version.minor >= 12 then
+    vim.notify("Warning: Your Neovim version (" .. version_str .. ") may have removed vim.tbl_islist. Update Packer ASAP.", vim.log.levels.WARN)
+  end
+  
+  if nvim_version.minor >= 13 then
+    vim.notify("Warning: Your Neovim version (" .. version_str .. ") may have removed vim.lsp.start_client. Update Copilot ASAP.", vim.log.levels.WARN)
+  end
+end
+
+-- Create a user command to easily run updates
+vim.api.nvim_create_user_command("FixDeprecationWarnings", function()
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'vim'

(W113)